docs/workload-tutorial/hello_world/stage_2/plug-in.h

```diff
@@ -19,9 +19,14 @@
 #include "workload_processor.h"
 
 
-class HelloWorld : public WorkloadProcessor {
+namespace HelloWorld {
+
+const char* workload_id="hello-world";
+
+class Workload : public WorkloadProcessor {
 public:
-    IMPL_WORKLOAD_PROCESSOR_CLONE(HelloWorld)
+    // Replace $CLASS_NAME$ with an appropriate class name, e.g. HelloWorld
```

Comment:
There is no $CLASS_NAME$ variable here. So it can  be confusing

Reply:
Fixed in 3 files:
./docs/workload-tutorial/templates/plug-in.h
./docs/workload-tutorial/hello_world/stage_2/plug-in.h
./docs/workload-tutorial/hello_world/stage_1/plug-in.h